[E 250607 00:05:48 smartWebSocketV2:303] Error occurred during WebSocket connection: module 'websocket' has no attribute 'WebSocketApp'
[I 250607 00:08:05 angelone_ws:15] WebSocket opened.
[I 250607 00:08:05 angelone_ws:27] Tick received: {'subscription_mode': 1, 'exchange_type': 1, 'token': '3045', 'sequence_number': 16510801, 'exchange_timestamp': 1749206260000, 'last_traded_price': 81305, 'subscription_mode_val': 'LTP'}
[W 250607 00:27:06 smartWebSocketV2:318] Attempting to resubscribe/reconnect (Attempt 1)...
[E 250607 00:27:14 angelone_ws:30] WebSocket error: Connection closed
[W 250607 00:27:14 smartWebSocketV2:342] Connection closed due to max retry attempts reached.
[I 250607 00:27:30 angelone_ws:15] WebSocket opened.
[I 250607 00:27:30 angelone_ws:27] Tick received: {'subscription_mode': 1, 'exchange_type': 1, 'token': '3045', 'sequence_number': 16510801, 'exchange_timestamp': 1749206260000, 'last_traded_price': 81305, 'subscription_mode_val': 'LTP'}
[I 250607 10:55:26 angelone_ws:16] WebSocket opened.
[W 250607 11:13:07 smartWebSocketV2:318] Attempting to resubscribe/reconnect (Attempt 1)...
[E 250607 11:13:17 angelone_ws:38] WebSocket error: Connection closed
[W 250607 11:13:17 smartWebSocketV2:342] Connection closed due to max retry attempts reached.
[E 250607 11:13:17 angelone_ws:38] WebSocket error: Connection closed
[W 250607 11:13:17 smartWebSocketV2:342] Connection closed due to max retry attempts reached.
[E 250607 11:13:17 angelone_ws:38] WebSocket error: Connection closed
[W 250607 11:13:17 smartWebSocketV2:342] Connection closed due to max retry attempts reached.
[E 250607 13:27:51 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '606665'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002055BE51310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250607 13:30:12 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '010553'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000028FD70C1310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250607 13:32:39 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '511551'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001FECF921310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250607 13:35:39 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '830741'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001A681541310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250607 13:46:59 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '400375'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000012979F81310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
[E 250607 14:56:44 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': '50:91:e3:21:d3:09', 'Accept': 'application/json', 'X-PrivateKey': 'NuzOVu1i', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S295624', 'password': '5451', 'totp': '707761'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x0000022FBF161310>, 'Connection to apiconnect.angelone.in timed out. (connect timeout=7)'))
